Cargo.toml

```diff
@@ -25,7 +25,7 @@ typst-render = { path = "crates/typst-render", version = "0.11.0" }
 typst-svg = { path = "crates/typst-svg", version = "0.11.0" }
 typst-syntax = { path = "crates/typst-syntax", version = "0.11.0" }
 typst-timing = { path = "crates/typst-timing", version = "0.11.0" }
-typst-assets = "0.11.0"
+typst-assets = { git = "https://github.com/Heinenen/typst-assets", branch = "translation" }
```

Comment:
I think the translation files are more like sources rather than the binary assets in typst-assets. It's useful to be able to update them more easily and also alongside normal PRs. This is also in line with [what rustc does](https://github.com/rust-lang/rust/blob/a7e3b1c8c5964113d0866b11114cbdaffd779a2e/compiler/rustc_middle/messages.ftl). I'd say you can just locate them under `crates/typst/translations/*.txt`.

Reply:
Yes definitely keep the source and strings versioned together in the same repo.